apps/web/client/src/app/project/[id]/_components/right-panel/chat-tab/chat-messages/message-content/tool-call-simple.tsx

```
@@ -69,54 +69,54 @@ export function ToolCallSimple({
 
     const getLabel = () => {
         try {
-            switch (toolName as keyof typeof TOOL_ICONS) {
+            switch (toolName) {
                 case TERMINAL_COMMAND_TOOL_NAME:
                     return 'Terminal';
                 case SEARCH_REPLACE_EDIT_FILE_TOOL_NAME:
                     const params = toolInvocation.args as z.infer<typeof SEARCH_REPLACE_EDIT_FILE_TOOL_PARAMETERS>;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)